src/App.scss

```diff
@@ -1 +1,54 @@
-// styles go here
+* {
+  // box-sizing: border-box;
+  justify-content: center;
+  align-items: center;
+  display: flexbox;
```

Comment:
Bad idea add display flexbox for every element

Reply:
done)